commands/translate.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const readline = require('readline');
+const { appFolder } = require('../constants');
+const readStream = fs.createReadStream([appFolder, 'file.dat'].join("/"));
+
+exports.command = 'translate [key]';
+exports.describe = 'Get the Garo Translation of the word';
+exports.handler = (argv) => {
+  const { key } = argv;
+  if(!key) {
+    console.error('Please provide a word to translate');
+  }
+
+  const rl = readline.createInterface({
+    input: readStream,
+    crlfDelay: Infinity,
+  });
+  
+  rl.on('line', (line) => {
```

Comment:
I believe that once a key is found, the parsing should stop. So, figure out a way to not have to parse the entire file every time!

Reply:
Done. Turns out that the stream creates a buffer, so its not going line by line, instead each chunk has 4 lines

Reply:
Ah that is a nice thing to know. You have not committed the change though right? For stopping parsing once a key is found?

Reply:
Just done that